build.gradle

```diff
@@ -196,17 +196,21 @@ task fastbuild(dependsOn: [
     ':dbfit-java:hsqldb:integrationTest'
     ])
 
-task integrationBuild(dependsOn: [
-    ':dbfit-java:db2:integrationTest',
-    ':dbfit-java:derby:integrationTest',
-    ':dbfit-java:hsqldb:integrationTest',
-    ':dbfit-java:mysql:integrationTest',
-    //':dbfit-java:netezza:integrationTest',
-    ':dbfit-java:oracle:integrationTest',
-    ':dbfit-java:postgres:integrationTest'
-    //':dbfit-java:sqlserver:integrationTest',
-    //':dbfit-java:teradata:integrationTest',
-    ])
+def integrationBuildTasks = []
```

Comment:
What if we just have a pure function returning the list of tasks? It's a bit harder to follow the side effects when we have kind of global variables which are later amended.